insights/attachments/links_domains.yml

```
@@ -1,12 +1,30 @@
 name: "Domains in attachments"
 type: "query"
 source: |
-  filter(
-    map(attachments,
-        map(file.explode(.),
-            distinct(map(.scan.url.urls, .domain.domain), .)
-        )
-    ),
-    length(.) > 0
+  filter(map(attachments,
+           map(file.explode(.),
+               distinct(map(filter(.scan.url.urls,
+                                   .domain.root_domain not in $tranco_1m
+                                   and .domain.root_domain not in $org_domains
```

filtering out tranco is not wanted on this one, that only applies to the low reputation insight
```suggestion
                                   .domain.root_domain not in $org_domains
```